portlet-tck_3.0/driver/src/main/java/javax/portlet/tck/driver/TCKSimpleTestDriver.java

```diff
@@ -228,15 +233,21 @@ public static void setUpBeforeClass() throws Exception {
       System.out.println("   PasswordId   =" + passwordId);
       System.out.println("   Browser      =" + browser);
       System.out.println("   Driver       =" + wd);
+      System.out.println("   binary       =" + binary + " ... used for ChromeDriver & FirefoxDriver only, for now");
 
       if (browser.equalsIgnoreCase("firefox")) {
-         driver = new FirefoxDriver();
+         driver = new FirefoxDriver(new FirefoxBinary(new File(binary)), new FirefoxProfile());
```

Comment:
Um ... i thought this was done, but I may need to change the property names.  So please wait again on this.